libcaf_bb/caf/bb/stream_reader.hpp

```diff
@@ -39,24 +39,24 @@ struct stream_reader_state {
     // nop
   }
 
-  void init(std::unique_ptr<InputStream> src_stream) {
-    stream = std::move(src_stream);
+  void init(std::unique_ptr<InputStream> input) {
+    this->input = std::move(input);
   }
 
   // -- properties -------------------------------------------------------------
 
   size_t at_end() const {
-    return !(*stream);
+    return !(*input);
   }
 
   // -- member variables -------------------------------------------------------
 
   /// Gives this actor a useful name in CAF logs.
   const char* name;
 
-  /// Stream we are about to stream
+  /// Input stream we are about to stream
```

Comment:
The comment is still hard to follow. The question is: What is this member and what are we doing with it? I'd suggest something along the lines of:

```
/// User-defined data source for reading ASCII or UTF-8 input line by line.
```